### PR TITLE
Cards component created successfully

### PR DESCRIPTION
**Pull request by ivinayakg**
Web capture_31-1-2022_221851_127 0 0 1

### CODE REVIEW

File: COMPONENTS/cards/index.css [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+  margin-left: auto;
+  display: flex;
+  justify-content: space-between;
+  gap: 10px;
+}
+
+.card_btnBox_box2 > * {
+  font-size: 1rem;
+  background-color: transparent;
+  color: gray;
```
Comment (chanchal16) [inline COMPONENTS/cards/index.css:100]:
```
Suggestion: You can use variables for colors. The rest looks good!
```

File: COMPONENTS/cards/index.css [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+.card_btnBox_box1 > * {
+  color: blueviolet;
+  font-weight: 500;
+  font-size: 1rem;
+  background-color: transparent;
+}
+
+.card_btnBox_box2 {
+  margin-left: auto;
+  display: flex;
```
Comment (chanchal16) [inline COMPONENTS/cards/index.css:92]:
```
Suggestion: You can create a generic class for flex as these properties are repeating for card btnbox_box.
```

Comment (ivinayakg) [reply COMPONENTS/cards/index.css:92]: i have created utility classes its just that in components I want them to be full by themselves as I would be needing to add ".flex" class in the container where I will use this class.
Thank you for your time.

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to me!

**Pull request merged**